src/components/memorygame/gameHelpers.jsx

```diff
@@ -0,0 +1,46 @@
+// manipulates isFlipped and isMatched property of card a state
+// property example : "isMatched"
+// cardStates is an array of objects(states)
+// see src/memorygame/MemoryGameBoard/index.jsx to for cardStates example and use of this function
+// targetCard is an element of cardStates array.
+// changeCardStatePropertyToOpposite is called when selectedCards state change.
+export const changeCardStatePropertyToOpposite = (
```

Comment:
invertCardStateForKey 

we call them keys usually. bc objects are key value pairs

Reply:
makes sense PTAL